kdc/kerberos5.c

```diff
@@ -2185,6 +2194,7 @@ _kdc_as_rep(astgs_request_t r)
 			pat[n].name, r->cname);
 		found_pa = 1;
 		r->et.flags.pre_authent = 1;
+		r->validated_pa_type = pat[n].type;
```

Comment:
If `found_pa` is now equivalent to `pa_type_validated != KRB5_PADATA_NONE`, then it should be removed. DRY.

Reply:
Thanks @josephsutton1 for fixing this up.